src/ontology/cl-edit.owl

```diff
@@ -27808,6 +27815,72 @@ AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_4023154 "CL:0000328")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023161 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023161 "unipolar brush cell")
+SubClassOf(obo:CL_4023161 obo:CL_0000099)
+SubClassOf(obo:CL_4023161 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070025))
+SubClassOf(obo:CL_4023161 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002028))
+SubClassOf(obo:CL_4023161 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061535))
+
+# Class: obo:CL_4023162 (bushy cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Bushy_cell&oldid=879061753"^^xsd:string) obo:IAO_0000115 obo:CL_4023162 "A neuron found in the anterior part of the ventral cochlear nucleus that has the appearance of bushes, having short dendrites. Bushy cells give outputs to different parts of the superior olivary complex.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023162 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023162 "bushy cell")
+SubClassOf(obo:CL_4023162 obo:CL_0000540)
+SubClassOf(obo:CL_4023162 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002830))
+SubClassOf(obo:CL_4023162 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0002128))
+
+# Class: obo:CL_4023163 (spherical bushy cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31881018"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Bushy_cell&oldid=879061753"^^xsd:string) obo:IAO_0000115 obo:CL_4023163 "A bushy cell that receives only few large excitatory endbulb synapses from auditory nerves. Sperhical bush cells give excitatory input to the lateral and medial parts of the superior olive.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023163 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31881018"^^xsd:string) Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasExactSynonym obo:CL_4023163 "SBC")
+AnnotationAssertion(rdfs:label obo:CL_4023163 "spherical bushy cell")
+SubClassOf(obo:CL_4023163 obo:CL_4023162)
+
+# Class: obo:CL_4023164 (globular bushy cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31881018"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Bushy_cell&oldid=879061753"^^xsd:string) obo:IAO_0000115 obo:CL_4023164 "A bushy cell that receives a larger number of medium-sized synapses, called modified endbulbs. Globular bushy cells extend to the superior olive on both sides of the brainstem where they give input to the bipolar neurons.")
```

Comment:
In definition, should "larger" be "large"? Otherwise, it is larger compared to what? Should "Globular bushy cells extend" be changed to singular form?

Reply:
yes large is right, sorry, think I copied it directly from source that would have context